# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1308671313 25200
#      Tue Jun 21 08:48:33 2011 -0700
# Branch stable
# Node ID 684ff193ddd23b36654e29e5baeef4f32f5b001e
# Parent  13bcd62824a74473cb77434cc81d9345335b696d
Restore ability to compile on Windows platforms

src/Makefile.am: Restore liboctave dependencies accidentally deleted in
changeset 633ef00d963c.

diff --git a/src/Makefile.am b/src/Makefile.am
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -570,16 +570,25 @@ OCTINTERP_LINK_DEPS = \
   $(READLINE_LIBS) $(TERM_LIBS) \
   $(LIBGLOB) \
   $(LAPACK_LIBS) $(BLAS_LIBS) \
   $(LIBS) \
   $(FLIBS)
 
 liboctinterp_la_LIBADD = $(OCTINTERP_LINK_DEPS)
 
+## Additional library dependencies used by module.mk files
+OCT_LINK_DEPS = \
+  $(RLD_FLAG) $(LDFLAGS) \
+  ./liboctinterp.la \
+  ../liboctave/liboctave.la \
+  ../libcruft/libcruft.la \
+  ../libcruft/libranlib.la \
+  ../libgnu/libgnu.la
+
 liboctinterp_la_CPPFLAGS = @OCTINTERP_DLL_DEFS@ $(AM_CPPFLAGS)
 
 # Increment these as needed and according to the rules in the libtool manual:
 liboctinterp_current = 0
 liboctinterp_revision = 0
 liboctinterp_age = 0
 
 liboctinterp_version_info = $(liboctinterp_current):$(liboctinterp_revision):$(liboctinterp_age)
